src/leases/helpers.js

```diff
@@ -150,6 +150,22 @@ export const getFullAddress = (item: Object): ?string => {
   return `${item.address || ''}${(item.postal_code || item.city) ? ', ' : ''}${item.postal_code ? item.postal_code + ' ' : ''}${item.city || ''}`;
 };
 
+/**
+ * Get a lease history item title text that is truncated when it is long
+ * @param {?string} text
+ * @param {number} MAX_TITLE_LENGTH
+ * @returns {string}
+ */
+
+export const getTitleText = (text: ?string, maxLength: number) => {
+  if (text) {
+    return text.length > maxLength ? text.substr(0, maxLength) + "..." : text
+  } else {
```

Comment:
Wondering the purpose of this else, could the whole if-else be removed and use `return text.length > maxLength ? text.substr(0, maxLength) + "..." : text`

Reply:
That ternary won't do, because if text is undefined, then the function would return a null. But I want this function to always return a string, empty or not. And I added the if-else statement so that I wouldn't need to include the null check in the ternary statement.